codeclimate_test_reporter/components/runner.py

```diff
@@ -38,11 +38,11 @@ def run(self):
         """
 
         try:
-            if self.parsed_args.debug:
-                self.out.write(self.__debug_info())
-                return 0
-            elif self.parsed_args.version:
-                self.out.write(reporter_version)
+            if self.parsed_args.version:
+                if self.parsed_args.debug:
```

Comment:
I guess it doesn't have to be. Main change is to not exit right away and continue processing. I'll move this back up and just remove the exit.